scipy/special/add_newdocs.py

```diff
@@ -1225,12 +1225,24 @@ def add_newdoc(place, name, doc):
     """
     gammaln(z)
 
-    Logarithm of absolute value of gamma function
+    Performs a logarithmic transformation of the
+    values of the gamma function in one of two
+    ways, depending on the input `z`:
 
-    Defined as::
+    1) type(z) != <type 'complex'>
```

Comment:
I'd suggest wording this like so:

```
1) `z` is not complex:
...
```

Strictly speaking, `type(z)` is likely `ndarray`! It's the `dtype` that matters.